docs/mjsonwp/protocol-methods.md

```diff
@@ -109,8 +109,8 @@ POST        | `/wd/hub/session/{sessionId}/appium/device/shake`
 POST        | `/wd/hub/session/{sessionId}/appium/device/lock`                       | Lock the device.
 POST        | `/wd/hub/session/{sessionId}/appium/device/unlock`                     | Unlock the device.
 POST        | `/wd/hub/session/{sessionId}/appium/device/is_locked`                  | Check whether the device is locked or not.
-POST        | `/wd/hub/session/{sessionId}/appium/performanceData/types`             | returns the information types of the system state which is supported to read as like cpu, memory, network traffic, and battery.
-POST        | `/wd/hub/session/{sessionId}/appium/getPerformanceData`				 | returns the information of the system state which is supported to read as like cpu, memory, network traffic, and battery.
+POST        | `/wd/hub/session/{sessionId}/appium/startRecordingScreen`              | start recording the screen.
```

Comment:
I apologize for the confusion. I meant that the performance data stuff was in the pull request. The pull request was requesting to re-do the performance data code. This is wrong. You need to start a new pull request, starting from the latest in master for the `appium-android-driver`, so that you are just adding the code for recording the screen, not the code for already-merged stuff and then the new code.

Otherwise we will not be able to merge it without a lot of working trying to untangle what has happened since that original code came in.

So... on your computer, do something like:
```
cd appium-android-driver
git checkout master
git pull origin master
git checkout -b heeseon-record
```
And then do the work of adding the new functions.